bot/src/main/java/com/dongtronic/diabot/platforms/discord/commands/diabetes/ConversionSlashCommand.kt

```diff
@@ -0,0 +1,42 @@
+package com.dongtronic.diabot.platforms.discord.commands.diabetes
+
+import com.dongtronic.diabot.logic.diabetes.BloodGlucoseConverter
+import com.dongtronic.diabot.logic.diabetes.GlucoseUnit
+import com.dongtronic.diabot.platforms.discord.commands.SlashCommand
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent
+import net.dv8tion.jda.api.interactions.commands.OptionType
+import net.dv8tion.jda.api.interactions.commands.build.CommandData
+
+class ConversionSlashCommand : SlashCommand {
+    private val commandArgGlucose = "glucose"
+    private val commandArgUnit = "unit"
+
+    override val commandName: String = "convert"
+
+    override fun config(): CommandData {
+        return CommandData(commandName, "Convert blood glucose values between mmol/L and mg/dL")
+                .addOption(OptionType.NUMBER, commandArgGlucose, "Blood glucose level", true)
+                .addOption(OptionType.STRING, commandArgUnit, "Blood glucose unit (mmol/L, mg/dL)")
```

Comment:
```suggestion
                .addOptions(
                        OptionData(OptionType.STRING, commandArgUnit, "Blood glucose unit (mmol/L, mg/dL)")
                                .addChoice("mmol/L", "mmol/L")
                                .addChoice("mg/dL", "mg/dL")
                )
```

This would probably make it easier for users by making them choose a unit rather than type it